Mage.Sets/src/mage/cards/s/SilverquillLecturer.java

```diff
@@ -0,0 +1,48 @@
+package mage.cards.s;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.continuous.GainAbilityControlledSpellsEffect;
+import mage.abilities.keyword.DemonstrateAbility;
+import mage.constants.SubType;
+import mage.filter.common.FilterNonlandCard;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class SilverquillLecturer extends CardImpl {
+
+    private static final FilterNonlandCard filter = new FilterNonlandCard("creature spells");
```

Comment:
Why nonland? Does it work with Zoetic Cavern casted as a morph.

Reply:
I have no idea, didn't know that card existed. I copied [[MycosynthGolem]]

Reply:
I'll check it in a bit and update if needed

Reply:
It works with Zoetic Cavern though I don't know if it does correctly if you copy it you get the land rather than the morph

Reply:
Alright that seems to be the way we add such "spell ability" on the card before being cast.
I think nonland is mostly there to avoid the ability showing on the lands in your hand, and since land can not be cast usually, if you say it works for Zoetic Cavern that's good enough for me.